internal/postprocessor/Dockerfile

```diff
@@ -39,8 +33,6 @@ RUN go install honnef.co/go/tools/cmd/staticcheck@latest && \
     go install golang.org/x/lint/golint@latest && \
     go install golang.org/x/tools/cmd/goimports@latest
 
-# Copy the binary to the production image from the builder stage.
-# post_processor is like an app
-COPY --from=builder /postprocessor/post_processor /post_processor
+# ENV GOCACHE="/tmp/"
```

Comment:
nit: remove this comment if it is not needed.

Reply:
done